src/Components/Web.JS/src/Platform/Circuits/CircuitManager.ts

```diff
@@ -91,15 +91,13 @@ function getComponentStartComment(node: Node): StartComponentComment | undefined
           return {
             node: node as Comment,
             circuitId,
-            rendererId: rendererId,
-            componentId: componentId,
+            rendererId: Number.parseInt(rendererId),
+            componentId: Number.parseInt(componentId),
           };
-        } else {
-          throw new Error(`Found malformed start component comment at ${node.textContent}`);
         }
       } catch (error) {
-        throw new Error(`Found malformed start component comment at ${node.textContent}`);
```

Comment:
Another minor thing here, the catch block would be hit if like 98 threw. I just moved this to a single throw statement for clarity.